sqlite3/assets/wasm/sqlite_cfg.h

```diff
@@ -44,7 +44,6 @@
 #define SQLITE_OMIT_AUTHORIZATION
 #define SQLITE_UNTESTABLE
 #define SQLITE_OMIT_COMPILEOPTION_DIAGS
-#define SQLITE_OMIT_LOAD_EXTENSION
```

Comment:
I don't think we can really use `dlopen` with wasm either way. So it can probably stay disabled if it keeps the size of the compiled code smaller.